src/System.Diagnostics.StackTrace/src/project.json

```diff
@@ -5,7 +5,11 @@
         "Microsoft.TargetingPack.Private.CoreCLR": "1.1.0-beta-24415-03",
         "System.IO.FileSystem": "4.0.1",
         "System.Reflection.Metadata": "1.4.1-beta-24415-01",
-        "System.Collections.Immutable": "1.2.0"
+        "System.Collections.Immutable": "1.2.0",
+        "System.Runtime.InteropServices": {
+          "version": "4.1.0",
+          "exclude": "compile"
```

Comment:
Another option here would have been to add a project reference to the facade like you did for others @https://github.com/dotnet/corefx/pull/10812/files#diff-561d3a2f14b5516bcd52f1f328cd688bR44.